ecosystem/node-checker/src/configuration/types.rs

```diff
@@ -102,6 +103,9 @@ pub struct EvaluatorArgs {
 
     #[clap(flatten)]
     pub consensus_evaluator_args: ConsensusMetricsEvaluatorArgs,
+
+    #[clap(flatten)]
+    pub build_version_evaluator_args: BuildVersionEvaluatorArgs,
```

Comment:
nit: we may want to consider alphabetical ordering if this list continues to grow.